Workbooks/Azure Active Directory TroubleShooting/SensitiveOperations/SensitiveOperations.workbook

```diff
@@ -0,0 +1,953 @@
+                  }
+                },
+                {
+                  "id": "e7a8c646-6800-40c4-9c21-445384a6a5bc",
+                  "version": "KqlParameterItem/1.0",
+                  "name": "Operation",
+                  "type": 2,
+                  "isRequired": true,
+                  "multiSelect": true,
+                  "quote": "'",
+                  "delimiter": ",",
+                  "value": [
+                    "Add member to role"
+                  ],
+                  "typeSettings": {
+                    "additionalResourceOptions": [
+                      "value::all"
+                    ],
+                    "showDefault": false
+                  },
+                  "jsonData": "[\r\n    {\"value\": \"Add member to role\", \"label\": \"Add member to role\"},\r\n    {\"value\": \"Add eligible member to role\", \"label\": \"Add eligible member to role\"},\r\n    {\"value\": \"Add scoped member to role\", \"label\": \"Add scoped member to role\"},\r\n    {\"value\": \"Add member to group\", \"label\": \"Add member to group\"}\r\n]",
+                  "timeContext": {
+                    "durationMs": 86400000
+                  }
+                },
+                {
+                  "id": "16bfa3f9-044a-4c17-b5a4-4116acc7a725",
+                  "version": "KqlParameterItem/1.0",
+                  "name": "InitiatingUserOrApp",
+                  "type": 1,
+                  "value": "All",
+                  "timeContext": {
+                    "durationMs": 86400000
+                  }
+                }
+              ],
+              "style": "pills",
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces"
+            },
+            "name": "parameters - 2"
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "AuditLogs\r\n| where TimeGenerated {TimeRange:value}\r\n| extend InitiatingUserOrApp = iff(isnotempty(InitiatedBy.user.userPrincipalName),tostring(InitiatedBy.user.userPrincipalName), tostring(InitiatedBy.app.displayName))\r\n| where InitiatingUserOrApp contains \"{InitiatingUserOrApp:escape}\" or \"{InitiatingUserOrApp:escape}\" == \"All\"\r\n| extend type = tostring(TargetResources[0].type)\r\n| where type == \"ServicePrincipal\"\r\n| where \"{Operation}\" has OperationName \r\n| project TimeGenerated, InitiatingUserOrApp, OperationName, ServicePrincipalDisplayName = TargetResources[0].displayName, GroupOrRoleNameAddedTo = TargetResources[0].modifiedProperties[1].newValue\r\n",
+              "size": 0,
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces",
+              "crossComponentResources": [
+                "{Workspace}"
+              ],
+              "sortBy": []
+            },
+            "name": "query - 1"
+          }
+        ]
+      },
+      "name": "group - 6"
```

Comment:
Do a quick scan of names like "query - ", "text - ", "parameters - ", etc... and make sure they are unique, this will allow for localization of these names